Samplero/Samplero/Screen/EstimateHistory/View/Cell/EstimateHistoryCollectionViewCell.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  EstimateHistoryCollectionViewCell.swift
+//  Samplero
+//
+//  Created by JiwKang on 2022/10/10.
+//
+
+import UIKit
+
+import SnapKit
+
+class EstimateHistoryCollectionViewCell: BaseCollectionViewCell {
+    
+    // MARK: - Properties
+    
+    private let imageView: UIImageView = {
+        let imageView: UIImageView = UIImageView()
+        return imageView
+    }()
```

Comment:
p5.
```suggestion
    private let imageView = UIImageView()
```

위와같이 선언해주면 될 것 같습니다.

Reply:
습관은 참 무섭습니다.
아무것도 안할건데 왜 저렇게 했을까...